requirements.dev.txt

```diff
@@ -4,6 +4,7 @@ colorama==0.4.4
 dpytest==0.5.3
 isort==5.10.1
 pytest-asyncio==0.16.0
+pytest-cov==3.0.0
```

Comment:
I added this out of curiousity. By running `pytest --cov=poglink tests/` you produce the following output which shows us we currently have 70% test coverage in `poglink`:

```
❯ pytest --cov=poglink tests/
======================================================================================================== test session starts ========================================================================================================
platform linux -- Python 3.7.7, pytest-6.2.5, py-1.11.0, pluggy-1.0.0
rootdir: /home/travipross/git/personal/poglink
plugins: asyncio-0.16.0, httpserver-1.0.2, cov-3.0.0
collected 57 items                                                                                                                                                                                                                  

tests/test_httpserver_dynamic.py ....                                                                                                                                                                                         [  7%]
tests/integration/test_async_http_request.py ..                                                                                                                                                                               [ 10%]
tests/integration/test_cogs_bans.py .                                                                                                                                                                                         [ 12%]
tests/integration/test_cogs_rates.py .....                                                                                                                                                                                    [ 21%]
tests/unit/test_bot.py ......                                                                                                                                                                                                 [ 31%]
tests/unit/test_cli.py ..                                                                                                                                                                                                     [ 35%]
tests/unit/test_config.py .                                                                                                                                                                                                   [ 36%]
tests/unit/test_models_bans.py ....                                                                                                                                                                                           [ 43%]
tests/unit/test_models_rates.py ...........                                                                                                                                                                                   [ 63%]
tests/unit/test_utils.py .....................                                                                                                                                                                                [100%]

----------- coverage: platform linux, python 3.7.7-final-0 -----------
Name                         Stmts   Miss  Cover
------------------------------------------------
poglink/__init__.py              6      2    67%
poglink/bot.py                  56      7    88%
poglink/cogs/__init__.py         1      0   100%
poglink/cogs/bans.py            56     27    52%
poglink/cogs/rates.py          111     25    77%
poglink/cogs/test.py            16     16     0%
poglink/config.py               41      7    83%
poglink/error.py                 8      0   100%
poglink/main.py                 58     58     0%
poglink/models/__init__.py       2      0   100%
poglink/models/bans.py          43      3    93%
poglink/models/rates.py         72      4    94%
poglink/utils.py                31      0   100%
------------------------------------------------
TOTAL                          501    149    70%
```

Reply:
If you want this set up with github actions and have support for shields/badges showing the coverage status, you'd just need to create a free account on [codecov](https://about.codecov.io/language/python/) and invite me to admin it. Would just need to trigger a github action to run this in the CI and publish the results to codecov.

Reply:
I've linked it to this repo. Your permissions here may actually apply on Codecov, but not sure.

Reply:
Did you follow [this](https://docs.codecov.com/docs#basic-usage)?

Reply:
Codecov upload is working. resolved.